guides/common/modules/con_content-flow-in-projectname.adoc

```diff
@@ -0,0 +1,28 @@
+[id="Content-Flow-in-{ProjectNameID}_{context}"]
+= Content Flow in {ProjectName}
+
+Content flow in {ProjectName} involves management and distribution of content from external sources to the managed systems.
+
+Content in {Project} flows from _external content sources_ to _{ProjectServer}_.
+_{SmartProxyServers}_ mirror the content to _managed hosts_.
+
+*External content sources*::
+You can configure many content sources with {Project}.
+The supported content sources include the Red{nbsp}Hat Customer Portal, Git repositories, Ansible collections, Docker Hub, Puppet Forge, SCAP repositories, or internal data stores of your organization.
```

Comment:
I'm not sure if Satellite can directly consume content from RH CP, but I'm missing a mention of RH CDN + link to the CDN structure for more info.

Reply:
A module on CDN is included in Managing Content, so I added a link to that guide.